config/crd/bases/controlplane.cluster.x-k8s.io_k3kcontrolplanes.yaml

```
@@ -150,6 +150,11 @@ spec:
                 - secretName
                 - secretNamespace
                 type: object
+              hostTargetNamespace:
+                description: |-
+                  HostTargetNamespace is the host namespace where the virtual cluster will be installed to.
```

```suggestion
                  HostTargetNamespace is the host namespace where the virtual cluster will be installed.
```
Maybe we can drop the proposition at the end of the sentence.